### ISSUE TITLE
A switch group is allowed to be added without a type

### ISSUE THREAD

**Issue by jrouzierinverse** [inverse-inc/packetfence#1108]
A switch group is allowed to be added without a type.
However when doing a checkup it throws an error.

How to replicate.
Go to 
Admin GUI > Configuration>Switches> Switch Groups
Click on 'Add Switch Group' and create a switch group without a type.

**Comment by julsemaan** [comment]
The default switch has a mandatory type and adding it to the default section is part of the upgrade guide.

**Comment by jrouzierinverse** [comment]
I tried that and the error still happens.
The issue seems to be happens the default argument was removed.
And the checkup code was expecting it to be there.

**Comment by julsemaan** [comment]
Can you post your switches.conf to reproduce ?

Thanks

**Comment by jrouzierinverse** [comment]
[switches.conf.txt](https://github.com/inverse-inc/packetfence/files/81403/switches.conf.txt)

**Comment by julsemaan** [comment]
The checkup works without giving me a warning on the type on my side : 

```
#bin/pfcmd checkup
Checking configuration sanity...
WARNING - Action 'action0' of rule 'file1 rule admins' is not part of the 'authentication' rule class allowed actions. It will be ignored.
WARNING - You defined a inline network (172.21.156.0/24) but no inline interface.
WARNING - invalid parameter billing_engine for profile bob20
WARNING - invalid parameter billing_engine for profile default
```

**Comment by jrouzierinverse** [comment]
Did you do a configreload?

**Comment by julsemaan** [comment]
I sure did

**Comment by extrafu** [comment]
Did you reboot?

**Comment by julsemaan** [comment]
@lzammit did for me 

He actually rebooted the ESX

**Comment by julsemaan** [comment]
Are you using the Macbook of @louismunro to test ?

**Comment by jrouzierinverse** [comment]
Why actually yes!
I found the issue I am checking to see if it was an issue with my setup or the code.

**Comment by jrouzierinverse** [comment]
To replicate
- ./bin/pfcmd cache configfiles clear
- Admin GUI > Configuration>Switches> default
- Click Save
- Then 'Perform Checkup'

**Comment by louismunro** [comment]
Zammit Corp.

Now [0] days without a reboot.

## 

Louis Munro
lmunro@inverse.ca  ::  www.inverse.ca 
+1.514.447.4918 x125  :: +1 (866) 353-6153 x125
Inverse inc. :: Leaders behind SOGo (www.sogo.nu) and PacketFence (www.packetfence.org)

**Comment by louismunro** [comment]
What should we do with this issue? 
Move to 5.7, close or else?

**Comment by jrouzierinverse** [comment]
This has been fixed in 850ec58abeea1f1efc12ac8841e25e6caa2ab84d
